src/components/pages/dashboard/dashboard.container.js

```diff
@@ -29,8 +29,8 @@ import { Dashboard } from './dashboard';
 
 const mapStateToProps = state => ({
   azureMapsKey: getAzureMapsKey(state),
-  azureMapsKeyError: getAzureMapsKeyError(state),
-  azureMapsKeyIsPending: getAzureMapsKeyPendingStatus(state),
+  azureMapsKeyError: getSolutionSettingsError(state),
```

Comment:
They will not. Since, AzureMapsKeys is now inside solutionSettings and dashboard page is only concerned about AzureMapsKeys and not the entire SolutionSettings.

Reply:
It feels weird, but I see what you're saying.